lib/resources.js

```diff
@@ -311,18 +318,21 @@ async function _getAcquisitionRecord({acquirerId}) {
         tokenized: [{
           tokenizerId,
           resources: {}
-        }]
+        }],
+        expires: new Date(),
+        ttl: 0
       }
     };
   }
+  // expose `expires` as milliseconds since the epoch
   if(record.acquisition.expires) {
     record.acquisition.expires = record.acquisition.expires.getTime();
   }
```

Comment:
Is this conditional still required?  I see expires is now part of the default record.  Is there a case when `record.acquisition.expires` is falsey?

Reply:
This was before the decision to require the collection to be dropped. I'll remove the conditional now.